### ISSUE TITLE
Embeded windows block events even to higher level CanvasLayer

### ISSUE THREAD

**Issue by cesarizu** [godotengine/godot#86840]
### Tested versions

4.0.4
4.1.3
4.2.1
4.3-dev1

### System information

Godot v4.2.1.stable - Windows 10.0.22631 - Vulkan (Mobile) - dedicated NVIDIA GeForce RTX 4070 Laptop GPU (NVIDIA; 31.0.15.3210) - 13th Gen Intel(R) Core(TM) i9-13900HX (32 Threads)

### Issue description

I have a custom software cursor that is set on a canvas layer and has some logic and presentation. When an embedded window is shown (e.g. popup window) the layer stops receiving events and the cursor is stuck in the last position.

Setting the `CanvasLayer`s `layer` to a number above `1024` is not possible in the inspector. Setting it through code makes the cursor appear on top of the popup but the events are still not delivered to the cursor.

Setting `mouse_passthrough` doesn't work (docs state that it only works for non-embedded windows and on some OSs only)

### Steps to reproduce

Create a canvas layer and listen for events on a children. Open a popup window and the events are no longer delivered to the canvas layer children.

### Minimal reproduction project (MRP)

[popup_test.zip](https://github.com/godotengine/godot/files/13844447/popup_test.zip)

**Comment by Sauermann** [comment]
This works as intended. Currently embedded Windows always block input events from other nodes in the embedder viewport.

I suggest to create a proposal for this in the proposal repository https://github.com/godotengine/godot-proposals.

**Comment by cesarizu** [comment]
Thanks. As a workaround/improvement, I've changed my popups to be just centered panels and they work just fine for my game.

I got this problem while porting my game to Godot 4. On Godot 3 they don't block input to the underlying canvas layers.

I understand that popups and other windows might be used for different purposes and that having the ability to block input is important.

Maybe, if I see the need in the future, I'll try to implement the `mouse_passthrough` functionality for embedded windows.
